gossip/src/crds_value.rs

```diff
@@ -204,6 +206,44 @@ impl SnapshotHash {
         }
     }
 }
+
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, AbiExample)]
+pub struct IncrementalSnapshotHash {
+    pub from: Pubkey,
+    pub base_slot: Slot,
+    pub hashes: Vec<(Slot, Hash)>,
+    pub wallclock: u64,
+}
+
+impl Sanitize for IncrementalSnapshotHash {
+    fn sanitize(&self) -> Result<(), SanitizeError> {
+        sanitize_wallclock(self.wallclock)?;
+        if self.base_slot >= MAX_SLOT {
+            return Err(SanitizeError::ValueOutOfBounds);
+        }
+        for (slot, _) in &self.hashes {
```

Comment:
Restricting `self.hashes.len()` to a sane value here seems nice (and I see we don't do this for `SnapshotHash` either!)

Reply:
What would be a good value. I assume I should just use `MAX_SNAPSHOT_HASHES`, right?

Reply:
Resolved here: https://github.com/solana-labs/solana/pull/20374#discussion_r721397268